3rdparty/compiler-rt/CMakeLists.txt

```diff
@@ -5,9 +5,9 @@ include(FetchContent)
 FetchContent_Declare(
   compiler-rt-sources
   SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/compiler-rt
-  URL https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.1/compiler-rt-10.0.1.src.tar.xz
+  URL https://github.com/llvm/llvm-project/releases/download/llvmorg-11.1.0/compiler-rt-11.1.0.src.tar.xz
```

Comment:
Does this also work with previous clang-10?

Reply:
The tests all pass when building with clang-10, so I think using v11.1.0 here works with clang-10.